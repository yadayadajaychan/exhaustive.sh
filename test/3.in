1A 3B
