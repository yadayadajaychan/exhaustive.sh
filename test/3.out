A=1'd0; B=3'd0; #10;
A=1'd0; B=3'd1; #10;
A=1'd0; B=3'd2; #10;
A=1'd0; B=3'd3; #10;
A=1'd0; B=3'd4; #10;
A=1'd0; B=3'd5; #10;
A=1'd0; B=3'd6; #10;
A=1'd0; B=3'd7; #10;
A=1'd1; B=3'd0; #10;
A=1'd1; B=3'd1; #10;
A=1'd1; B=3'd2; #10;
A=1'd1; B=3'd3; #10;
A=1'd1; B=3'd4; #10;
A=1'd1; B=3'd5; #10;
A=1'd1; B=3'd6; #10;
A=1'd1; B=3'd7; #10;
